.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,15 +1,12 @@
-Fixes #
+Fixes #<!-- Insert issue number that this PR fixes just after #-->  
 
 Changes made in this Pull Request:
- - 
+ - <!-- Describe the changes that this PR makes. --> 
```

Comment:
I'd remove, the "Changes made in this Pull Request" is already pretty clear, isn't it?

Reply:
It is, yet people leave this bullet point empty most of the time... I was an attempt to clarify that they should fill it.